src/core/unit.c

```diff
@@ -6379,30 +6379,40 @@ Condition *unit_find_failed_condition(Unit *u) {
         return failed_trigger && !has_succeeded_trigger ? failed_trigger : NULL;
 }
 
-int unit_can_live_mount(const Unit *u, sd_bus_error *error) {
+bool unit_can_live_mount(const Unit *u, sd_bus_error *error) {
```

Comment:
This sets variety of error code to `error`, but the argument is optional.
I'd like to keep this int, and propagate the error code.

Reply:
I was told to make it as such in the chat

Reply:
I read @YHNdnzj's opinion in the chat. I agree with that functions `foo_can_xyz)` should return bool. But, that's if you name the function so. I'd like to rename the function to `unit_verify_live_mount()` or so.

Reply:
Why?

Reply:
See comment below https://github.com/systemd/systemd/pull/34175#discussion_r1737878117.